### PR TITLE
Show image that introduces concept of splits/configs

### LINKED ISSUE #1212: Add Parquet advantages in dataset viewer docs
In https://huggingface.co/docs/hub/datasets-viewer#access-the-parquet-files

See https://github.com/huggingface/datasets-server/issues/2420

### PR DESCRIPTION
**Pull request by severo**
Adds the gif:

https://github.com/huggingface/hub-docs/assets/1676121/4f6814c1-1e88-4eb7-b879-771dd328487f

also: fixes #1212 -> https://moon-ci-docs.huggingface.co/docs/hub/pr_1215/en/datasets-viewer. Is it too much?

Capture d’écran 2024-02-09 à 12 41 56

### CODE REVIEW

**Comment by severo** [comment]
ok bot

File: docs/hub/datasets-data-files-configuration.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ There are no constraints on how to structure dataset repositories.
 However, if you want the Dataset Viewer to show certain data files, or to separate your dataset in train/validation/test splits, you need to structure your dataset accordingly.
 Often it is as simple as naming your data files according to their split names, e.g. `train.csv` and `test.csv`.
 
+## What are splits and configurations?
+
+Machine learning datasets typically have splits and may also have configurations. A _split_ is a subset of the dataset, like `train` and `test`, that are used during different stages of training and evaluating a model. A _configuration_ is a sub-dataset contained within a larger dataset. Configurations are especially common in multilingual speech datasets where there may be a different configuration for each language. If you're interested in learning more about splits and configurations, check out the [conceptual guide on "Splits and configurations"](https://huggingface.co/docs/datasets-server/configs_and_splits)!
```
Comment (stevhliu) [inline docs/hub/datasets-data-files-configuration.md:10]:
```suggestion
Machine learning datasets typically have splits and may also have configurations. A _split_ is a subset of the dataset, like `train` and `test`, that are used during different stages of training and evaluating a model. A _configuration_ is a sub-dataset contained within a larger dataset. Configurations are especially common in multilingual speech datasets where there may be a different configuration for each language. If you're interested in learning more about splits and configurations, check out the [Splits and configurations](https://huggingface.co/docs/datasets-server/configs_and_splits) guide!
```

File: docs/hub/datasets-viewer.md [hunk 2]
```diff
@@ -37,6 +37,21 @@ In this case, an informational message lets you know that the Viewer is partial.
 
 To power the dataset viewer, the first 5GB of every dataset are auto-converted to the Parquet format (unless it was already a Parquet dataset). In the dataset viewer (for example, see [`datasets/glue`](https://huggingface.co/datasets/glue)), you can click on [_"Auto-converted to Parquet"_](https://huggingface.co/datasets/glue/tree/refs%2Fconvert%2Fparquet/cola) to access the Parquet files. Please, refer to the [Datasets Server docs](/docs/datasets-server/parquet_process) to learn how to query the dataset parquet files with libraries such as Polars, Pandas or DuckDB.
 
+<Tip>
```
Comment (stevhliu) [inline docs/hub/datasets-viewer.md:40]: Nice! I think if we want to use it as a `<Tip>` here, I would keep it shorter to maybe just 2-3 sentences and let the linked content do the heavy lifting of explaining in more detail the advantages :)

**Review by stevhliu — APPROVED** [review]
LGTM! I think the gif does a nice job of visually explaining it so I don't think it's too much :)

**Review by julien-c — COMMENTED** [review]
lgtm, maybe related: you could add a section with a screenshot of parquet-converter profile so it gives it "legitimacy"

image

File: docs/hub/datasets-data-files-configuration.md [hunk 3]
```diff
@@ -5,6 +5,12 @@ There are no constraints on how to structure dataset repositories.
 However, if you want the Dataset Viewer to show certain data files, or to separate your dataset in train/validation/test splits, you need to structure your dataset accordingly.
 Often it is as simple as naming your data files according to their split names, e.g. `train.csv` and `test.csv`.
 
+## What are splits and configurations?
+
+Machine learning datasets typically have splits and may also have configurations. A _split_ is a subset of the dataset, like `train` and `test`, that are used during different stages of training and evaluating a model. A _configuration_ is a sub-dataset contained within a larger dataset. Configurations are especially common in multilingual speech datasets where there may be a different configuration for each language. If you're interested in learning more about splits and configurations, check out the [Splits and configurations](https://huggingface.co/docs/datasets-server/configs_and_splits) guide!
```
Comment (lhoestq) [inline docs/hub/datasets-data-files-configuration.md:10]: We usually use the words "subset"/"sub-dataset" as aliases for configurations, so maybe we can avoid using this for splits.

```suggestion
Machine learning datasets typically have splits and may also have configurations. A dataset is generally made of _splits_ (e.g. `train` and `test`) that are used during different stages of training and evaluating a model. A _configuration_ is a sub-dataset contained within a larger dataset. Configurations are especially common in multilingual speech datasets where there may be a different configuration for each language. If you're interested in learning more about splits and configurations, check out the [Splits and configurations](https://huggingface.co/docs/datasets-server/configs_and_splits) guide!
```

**Review by lhoestq — APPROVED** [review]
Love the gif :)

File: docs/hub/datasets-viewer.md (hunk 2 above)
Comment (severo) [reply docs/hub/datasets-viewer.md:40]: done, thanks

**Comment by severo** [comment]
> you could add a section with a screenshot of parquet-converter profile so it gives it "legitimacy"

done

**Pull request merged**